src/world/message_server.cpp

```diff
@@ -308,6 +296,46 @@ void message_server_listen(const bool& requestExit)
     }
 }
 
+void cache_zone_settings()
+{
+    const char* query = "SELECT zoneid, zoneip, zoneport, misc FROM zone_settings;";
+    int         ret   = sql->Query(query);
+
+    if (ret == SQL_ERROR)
+    {
+        ShowError("Error loading zone settings from DB. World server will not send messages.");
+        return;
+    }
+
+    // Keep track of the zones, as well as a list of unique ip / port combinations.
+    std::set<uint64> zoneIppSet;
```

Comment:
I wonder if zoneIpp is the best name for this. Its what's being stored, sure, but it represents a ip/port pair, which is how we identify processes on a machine, each of which can hold multiple zones. I don't have a better name suggestion, but its worth mentioning.

Reply:
`mapEndpointSet`?

Reply:
Sure

Reply:
Renamed `zoneIpp` to `mapEndpoint` in all instances